perceval/backends/core/meetup.py

```diff
@@ -82,7 +81,7 @@ class Meetup(Backend):
         ['venue']
     ]
 
-    def __init__(self, group, api_token, is_oauth_token=False,
+    def __init__(self, group, api_token,
```

Comment:
Can you update the documentation of this class?  It should say it uses OAuth2 tokens. There's also a reference to `api_key` when the parameter is `api_token` actually.